RCode/NEFI/run_bayes_model.R

```diff
@@ -73,9 +78,26 @@ jags.out <- run.jags(model = model,
 write.jagsfile(jags.out, file=file.path("Results/Jags_Models/",paste(site,paste0(model_name,'.txt'), sep = '_')), 
                remove.tags = TRUE, write.data = TRUE, write.inits = TRUE)
 
-png(file=file.path("Results/Jags_Models/",paste(site,paste0(model_name,'_Convergence.png'), sep = '_')))
-plot(jags.out, vars = jags_plug_ins$variable.names.model) 
-dev.off()
+#this will create multiple plots if var names are different but doesn't create multiple
+#plots for vars with same names, i.e., betas, so have created a quick hack to fix that
+
+#ok, you have to edit this vector to include the actual names of the parameters in your model :(
+vars <- c("tau_add","beta[1]","tau_yr")
```

Comment:
Did you try `jags_plug_ins$variable.namesout.model` ? This should have parameters that are model specific.